google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -99,11 +116,6 @@ public OSProcessHandler createProcessHandler(
             executor.run();
 
             Process devappserverProcess = executor.getProcess();
-            if (devappserverProcess == null) {
```

Comment:
If that's actually the case that ExecutionException's don't get swallowed by the platform (and actually report and error) we need to fix some other code I think:

The sdk validation logic (empty sdk, invalid sdk) is also using this exception. In these cases we definitely should not be reporting errors.